recipes/libsafec/all/conanfile.py

```diff
@@ -99,6 +101,14 @@ def generate(self):
             f"--enable-strmax={self.options.strmax}",
             f"--enable-memmax={self.options.memmax}",
         ]
+
+        if is_apple_os(self) and self.settings.arch in ["x86_64", "armv8"]:
+            tc.configure_args += [
+                "--disable-hardening",
```

Comment:
looking at the comments upstream, this was back when apple clang did not support some compiler flag - does it build without this?

Reply:
Yes, I just checked it locally